source/core/ut_utils.pkb

```diff
@@ -328,5 +328,33 @@ create or replace package body ut_utils is
     end if;
   end;
 
+  function to_xpath(a_list varchar2) return varchar2 is
+    l_xpath varchar2(32767) := a_list;
+  begin
+    if l_xpath not like '/%' then
+      l_xpath := '//'||replace(l_xpath,',','|//');
```

Comment:
The CSV is the current behavior and I think we should.
But now that I look at it i think we need to be more clever or clearly state in the documentation that:
You can only use CSV list of column/attribute names to exclude or XPath expression. You cannot mix XPath with CSV.
Also not sure if we should use simple `//` to remove attribute/column anywhere in XML, or should we remove only top level when CSV list is used. Probably top-level only is more predictable and less error-prone.
So the to_xpath would need to change.